src/hashTable.js

```diff
@@ -0,0 +1,125 @@
+class Node {
+  constructor(key, value) {
+    this.key = key,
+    this.value = value,
+    this.next = null
+  }
+}
+
+export default class HashTable {
+  constructor() {
+    this._length = 0
+    this.valueStore = []
+  }
+
+  hashFunction(key) {
+    let sum = 0
+    for (let i = 0; i < key.length; i++) {
+      sum += key.charCodeAt(i)
+    }
+    return sum % 3550
+  }
+
+  put(key, value) {
+    let newNode = new Node(key, value)
+    let hash = this.hashFunction(key)
+
+    if(!this.valueStore[hash]) {
+      this.valueStore[hash] = newNode
+    } else {
+      this.chain(hash, newNode)
+    }
+    ++this._length
+  }
+
+  size() {
+    return this._length
+  }
+
+  get(key) {
+    let hash = this.hashFunction(key)
+    if (this.valueStore[hash]) {
+      let currentNode = this.valueStore[hash]
+      if (currentNode.key === key) {
+        return currentNode.value
+      } else {
+        while (currentNode.next) {
+          if (currentNode.key === key) {
+            return currentNode.value
+          } else {
+            currentNode = currentNode.next
+          }
+        }
+        if (currentNode.key === key) {
+          return currentNode.value
+        } else {
+          return "no value here"
```

Comment:
I would not recommend returning a string here, as this would not necessarily indicate failure to the caller.  What if strings are the values that are being held in this hash, and someone stored the string "no value here"? How would the client determine that this was a failure?

Consider returning `null` when a value is not found.